tests/console/snapper_create_from.pm

```diff
@@ -0,0 +1,70 @@
+# SUSE's openQA tests
+#
+# Copyright © 2020 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Test snapper create when branching from a different snapshot.
+# `snapper` is used for creating the snapshots used by transactional-update and
+# not only can create a snapshot of the currently active one, but also has available
+# the option `--from` to branch off a different snapshot.
+# Maintainer: Joaquín Rivera <jeriveramoya@suse.com>
+
+use strict;
+use warnings;
+use base "opensusebasetest";
+use testapi;
+use utils;
+use Test::Assert ':all';
+use scheduler 'get_test_suite_data';
+
+sub run {
+    select_console 'root-console';
+
+    my $snapshots = get_test_suite_data()->{snapshots};
+
+    my $last_printed_num;    # keep track of last snapshot created
+    my $last_uuid;           # keep track of last snapshot uuid
+    my $btrfs_info;
+
+    # process each snapshot according to test data
+    foreach my $snapshot (@{$snapshots}) {
+        record_info("snap", $snapshot->{description});
```

Comment:
you could move the record_info lower in the code and replace the "snap" (which doesnt give any info anyway) with more helpful information. the description is fine but i am thinking that you could use the uuid for example to distiguish the subvolumes.

Reply:
uuid is too long for the box to fit and snap is enough descriptive IMO for the number or characters that we can put there.

Reply:
it was just a suggestion. personally i dont like it but not a big deal